test/SlowTests/Tests/Views/MapReduce.cs

```diff
@@ -111,10 +113,18 @@ public void DoesNotOverReduce()
 
                     WaitForIndexing(store);
 
-                    var index = store.Admin.Send(new GetIndexStatisticsOperation("CommentsCountPerBlog"));
+                    IndexStats stats = null;
+
+                    Assert.True(SpinWait.SpinUntil(() =>
+                    {
```

Comment:
These really tend to kill our overall test performance. Any way we can do that with events?

Reply:
We could do similar as for ETL tests:
https://github.com/ravendb/ravendb/blob/v4.0/test/SlowTests/Server/Documents/ETL/EtlTestBase.cs#L60-L72

Reply:
Yes, that would be better